packages/models/src/models/AppLogsModel.ts

```diff
@@ -24,22 +24,35 @@ export class AppsLogsModel extends BaseRaw<any> implements IAppLogsModel {
 					'appId': 1,
 					'_updatedAt': -1,
 					'entries.severity': 1,
+					'instanceId': 1,
 					'method': 1,
 				},
-				name: 'appId_indexed_query',
+				name: 'appId_indexed_query_v2',
 			},
 			// Index for queries on general logs endpoint
 			{
 				key: {
 					'_updatedAt': -1,
 					'entries.severity': 1,
+					'instanceId': 1,
 					'method': 1,
 				},
-				name: 'general_logs_index',
+				name: 'general_logs_index_v2',
```

Comment:
For these changes, I'm considering adding a migration file that deletes the old indexes as housekeeping. It would not cause errors if we don't do it, but leaves the indexes in the collection in a "dirty" state